lib/parallel_tests/cli.rb

```diff
@@ -26,6 +26,16 @@ def run(argv)
 
     private
 
+    def handle_interrupt
+      Kernel.exit if @graceful_shutdown_attempted
```

Comment:
might want to set `@graceful_shutdown_attempted ||= false` to not have warnings if someone runs ruby with warnings

Reply:
Good point